themes/wilson_theme_starterkit/templates/content/node--article--horizontal-card.html.twig

```diff
@@ -0,0 +1,81 @@
+{#
+/**
+ * @file
+ * Theme override to display an article teaser node.
+ *
+ * Available variables:
```

Comment:
As a general rule, if a template extends/overrides another, I wouldn't expect to see the comments/available variables list from the original - only the list of variables being added in this current template.

If the comments are important you could always add `@see path/to/core/template.html.twig`, or if there are a couple of pertinent variables (eg you've added extra info to the `metadata` variable) then call them out specifically, but otherwise this just feels needlessly repetitive IMO. 

As ever, not a blocker, especially if there are other files that do the same already.

Reply:
Good suggestion. A change that could be included in a future PR. There might be quite a big update coming for the Drupal 10 version of Wilson so this is cleanup that could be targeted there.